src/Teams/Data/Repositories/IProgramCodeQuestionRepository.cs

```diff
@@ -9,5 +9,7 @@ namespace Teams.Data.Repositories
     public interface IProgramCodeQuestionRepository
     {
         ProgramCodeQuestion PickById(Guid id);
+        void SaveProgramText(ProgramCodeQuestion question, string text);
```

Comment:
Let's have a separate repository for this action, as it doesn't change ProgramCodeQuestion